docs/faq.md

```diff
@@ -232,14 +232,14 @@ updates:
 
 #### Option 2: Requirement files
 
-The second option, and the only one that supports other CI systems, is using a `requirements-*.txt` file. The file should have a distinct name and have only one entry:
+The second option, and the only one that supports other CI systems, is using a `.github/requirements-*.txt` file. The file should have a distinct name and have only one entry:
```

Comment:
This one might be an error, I think? it seems to contradict the 'other CI systems' previously.

Reply:
Oops, typo, added it to the wrong spot.

Reply:
Should be moved to the right place now.